Icon: Add css classes to allow styling from external stylesheets

This should be a fairly straight forward change and should not effect the UI or functionality of the component. It's what we'd consider a "no harm" change. Here's what you need to do:

- [ ] Fork the repo
- [ ] run `npm install`
- [ ] run `npm run demo`. This will open up a new browser window with demos of each component. It will refresh whenever you save changes to the files in the `src` directory.
- [ ] open up `src/components/Icon`
- [ ] On line 595, add `className='mx-icon'` on it's own line, before the `fit` prop declaration.
- [ ] Open the demo and make sure your changes didn't break the Icon component.
- [ ] run `eslint src` to check for syntax and style errors. Fix the errors as needed.
- [ ] Commit your changes to the `src/` directory
- [ ] Push your changes to your fork
- [ ] Create a PR and include `Fixes #37` in the message
- [ ] Get merged and celebrate :tada:

I am working on this issue and got a problem in the "run `eslint`" step: "`eslint` was not recognized as an internal or external command".

I installed the `eslint` npm package globally (`npm install -g eslint`) and, when running the command again, got a Node.js error:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'eslint-plugin-react'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at C:\Users\Guilherme\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:120:26
    at Array.forEach (native)
    at loadPlugins (C:\Users\Guilherme\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:111:21)
    at processText (C:\Users\Guilherme\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:198:5)
    at processFile (C:\Users\Guilherme\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:258:18)
    at executeOnFile (C:\Users\Guilherme\AppData\Roaming\npm\node_modules\eslint\lib\cli-engine.js:604:23)
```

Then wondered that `eslint-plugin-react` also needs to be installed globally. Did it.

Now running `eslint src\` gives the following result:

```
C:\git\danguilherme\mx-react-components\src\components\DonutChart.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\Icon.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\Loader.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\Modal.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\RajaIcon.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\RangeSelector.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\Select.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\Spin.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\ToggleSwitch.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\components\TypeAhead.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\constants\Style.js
  0:0  error  Cannot find module 'babel-eslint'

C:\git\danguilherme\mx-react-components\src\Index.js
  0:0  error  Cannot find module 'babel-eslint'

✖ 12 problems (12 errors, 0 warnings)
```

And now I installed `babel-eslint` globally.

Running the command gives the following output:

```
C:\git\danguilherme\mx-react-components\src\components\DonutChart.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\Icon.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\Loader.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\Modal.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\RajaIcon.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\RangeSelector.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\Select.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\Spin.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\ToggleSwitch.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\components\TypeAhead.js
  1:32  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\constants\Style.js
  1:19  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

C:\git\danguilherme\mx-react-components\src\Index.js
  1:19  error  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style

✖ 12 problems (12 errors, 0 warnings)
```

I hadn't touch the other files, so they were already cloned with CRLF (`git status` only brings `src/components/Icon.js` as modified). As I am using Atom, it was easy to change from CRLF to LF, but I am not sure if I should modify all of them. By now I modified only `Icon.js`.

When committing the changes, git showed the following message:
```
warning: LF will be replaced by CRLF in src/components/Icon.js.
The file will have its original line endings in your working directory.
```
I'm confused. Here's the [forked repo](https://github.com/danguilherme/mx-react-components).

---

Is the approach described above (about installing the global npm packages) correct? If yes, wouldn't it be better to add a step for it in the next "first-timers-only" issues?

--- a/src/components/Icon.js
+++ b/src/components/Icon.js
@@ -578,6 +578,7 @@ class Icon extends React.Component {
     return (
       <svg
         {...this.props}
+        className='mx-icon'
         fit={true}
         preserveAspectRatio='xMidYMid meet'
         style={[styles, this.props.style]}

